src/order/order.ts

```diff
@@ -60,3 +61,45 @@ export interface GiftCertificateOrderPayment extends OrderPayment {
         remaining: number;
     };
 }
+
+export interface OrderConsignment {
+    shipping: OrderShippingConsignment[];
+}
+
+export interface OrderShippingConsignment {
+    lineItems: Array<{
+        id: number;
+    }>;
+    shippingAddressId: number;
+    firstName: string;
+    lastName: string;
+    company: string;
+    address1: string;
+    address2: string;
+    city: string;
+    stateOrProvince: string;
+    postalCode: string;
+    country: string;
+    countryCode: string;
+    email: string;
+    phone: string;
+    itemsTotal: number;
+    itemsShipped: number;
+    shippingMethod: string;
+    baseCost: number;
+    costExTax: number;
+    costIncTax: number;
+    costTax: number;
+    costTaxClassId: number;
+    baseHandlingCost: number;
+    handlingCostExTax: number;
+    handlingCostIncTax: number;
+    handlingCostTax: number;
+    handlingCostTaxClassId: number;
+    shippingZoneId: number;
+    shippingZoneName: string;
+    customFields: Array<{
```

Comment:
Do the properties of this key be `{ name: string, value: string }`
Reference https://github.com/bigcommerce/api-specs/blob/master/reference/orders.sf.yml#L774

Reply:
Thanks for pointing this out. Could you please have another review?